src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_floor(building: Clinic, temperature):
+    """
+    Computes the transmission load of the floor in kWh for a building.
+
+    """
+
+    return (
+        building.floor_area
+        * building.u_value_floor
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def transmission_load_roof(building: Clinic, temperature):
+    """
+    Computes the transmission load of the roof in kWh for a building.
+
+    """
+
+    return (
+        building.roof_area
+        * building.u_value_roof
+        * (temperature - building.inside_ideal_temperature)
+        / 1000
+    )
+
+
+def calculate_transmission_load(building: Clinic, temperature):
+    """
+    Computes the total transmission heat load in kWh for a building.
+
+    """
+    t1 = transmission_load_walls(MY_CLINIC, temperature)
+    t2 = transmission_load_doors_windows(MY_CLINIC, temperature)
+    t3 = transmission_load_floor(MY_CLINIC, temperature)
+    t4 = transmission_load_roof(MY_CLINIC, temperature)
+
+    return t1 + t2 + t3 + t4
+
+
+def internal_load_people_1(building: Clinic):
+    """
+    Computes the internal heat load of the nurses in kWh for a building.
+
+    """
+    # load_p = np.where((time_people>=11)&(time_people<18), building.people_1 * 1 * building.heat_loss_people_1 / 1000., 0.)
+    people_at_times = [
+        [
+            [0, building.people_1 * building.heat_loss_people_1][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_1, building.end_time_people_1)
+    ]
+    return people_at_times
+
+
+def internal_load_people_2(building: Clinic):
+    """
+    Computes the internal heat load of the patients and accompainers in kWh for a building.
+
+    """
+    people_at_times = [
+        [
+            [0, building.people_2 * building.heat_loss_people_2][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_2, building.end_time_people_2)
+    ]
+    return people_at_times
+
+
+def calculate_internal_load_people(building: Clinic):
+    """
+    Computes the internal heat load of the total people in kWh for a building.
+
+    """
+    p1 = internal_load_people_1(MY_CLINIC)
+    p2 = internal_load_people_2(MY_CLINIC)
+
+    print(p1[i] + p2[i] for i in range(len(p1[0])))
+
+
+calculate_internal_load_people(MY_CLINIC)
+
+
+def internal_load_lighting(building: Clinic):
+    """
+    Computes the internal heat load of the lamps in kWh for a building.
+
+    """
+    lamps_at_times = [
+        [
+            [0, building.lamps_internal * building.wattage_lamps][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_lamps, building.end_time_lamps)
+    ]
+    return lamps_at_times
+
+
+def equipment_load(building: Clinic):
```

Comment:
### :memo: Rename
The name of this function suggests that it's calculating the load of the equipment in the health centre... but the function is just calculating the load due to the fridge...

### 📏 Extensibility
Do you just need the electrical load of the fridge?
* If so, you can, as per my suggestion above for lamps, use the existing CLOVER functionality here for electrical devices;
* If not, i.e., if you're wanting the heat output from the fridge, then potentially this isn't the best way to do this. While it will work for the fridge, it's not the most "extensible," i.e., it can't be easily extended to MRI machines etc., and can't be extended to look at scenarios where you add different devices/electrical loads, without coming into the code each time you want to make a change.